src/main/java/it/gov/pagopa/pu/classification/repository/AssessmentsRepository.java

```diff
@@ -27,14 +32,14 @@ public interface AssessmentsRepository extends JpaRepository<Assessments,Long> {
       AND (cast(:#{#updateDateTimeIntervalFilter.from} AS STRING) IS NULL OR a.updateDate >= :#{#updateDateTimeIntervalFilter.from})
       AND (cast(:#{#updateDateTimeIntervalFilter.to} AS STRING) IS NULL OR a.updateDate <= :#{#updateDateTimeIntervalFilter.to})
       AND (:iuv IS NULL OR ad.iuv = :iuv)
-      AND (:debtPositionTypeOrgCode IS NULL OR a.debtPositionTypeOrgCode = :debtPositionTypeOrgCode)
+      AND (:debtPositionTypeOrgCodes IS NULL OR a.debtPositionTypeOrgCode IN :debtPositionTypeOrgCodes)
       AND (:status IS NULL OR a.status = :status)
     """)
   Page<Assessments> findPagedAssessments(
     @Param("assessmentName") String assessmentName,
     @Param("updateDateTimeIntervalFilter") LocalDateTimeIntervalFilter updateDateTimeIntervalFilter,
     @Param("iuv") String iuv,
-    @Param("debtPositionTypeOrgCode") String debtPositionTypeOrgCode,
+    @Parameter(required = true, array = @ArraySchema(uniqueItems = true, schema = @Schema(type = "String"))) @Param("debtPositionTypeOrgCodes") Set<String> debtPositionTypeOrgCodes,
```

Comment:
this method is not exposed through SpringDataRest... this annotation should not be needed
```suggestion
    @Param("debtPositionTypeOrgCodes") Set<String> debtPositionTypeOrgCodes,
```